### PR TITLE
Add dev script and Dockerfile

### PR DESCRIPTION
**Pull request by jinroh**
This PR add a `devtool/dev.sh` script that can run a complete env to develop a cozy client-side application. Its deps are:
  - `go`
  - `git`
  - `curl`
  - `couchdb` (optional)

This script will launch by default a proxy to serve the specified application directory on `app.cozy.local:8080` and proxy the `cozy-stack` on `cozy.local:8080`

Usage:

```
Usage: ./devtool/dev.sh [-h] [-d <app path>] [–v <stack version>]

Environment variables

  COZY_DEV_HOST
    specify the hostname or domain on which the dev server is listening
    to incoming requests. default: cozy.local

  COZY_DEV_PORT
    specify the port on which the dev server is listening.
    default: 8080.

  COZY_STACK_PATH
    specify the path of the cozy-stack binary folder or the binary
    itself. default: $GOPATH/bin.

  COZY_STACK_HOST
    specify the hostname on which the cozy-stack is launched.
    default: localhost.

  COZY_STACK_PORT
    specify the port on which the cozy-stack is listening.
    default: 8080.

  COUCHDB_ENABLE
    specify whether or not this script should launch couchdb.
    default: 1

  COUCHDB_HOST
    specify the host of the couchdb database. default: localhost

  COUCHDB_PORT
    specify the port of the couchdb database. default: 5984
```

It also add a Dockerfile to encapsulate all the dependencies of this script.

~~NOTE: the documentation is not yet written, it's coming.~~

### CODE REVIEW

**Comment by nono** [comment]
We start having some scripts: one in this PR, one in #113 for code coverage, one for `build.sh`, etc. Maybe we should put them in a `scripts/` directory.

**Comment by jinroh** [comment]
@nono I moved our scripts in `scripts/` directory.

**Comment by jinroh** [comment]
I don't really known what's going on with travis...

File: docs/client-app-dev.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+Develop a client-side application
+=================================
+
+This document describe a tool to run an environment in order to develop client-side application on the cozy-stack.
+
+We provide two different to ways to run this environment, either manually where you have to install part of the dependencies yourself or *via* a Docker image in which all dependencies are packed.
```
Comment (nono) [inline docs/client-app-dev.md]: s/to ways/ways/

**Pull request merged**